blazor/diagram/connectors/segments/multiplesegments.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: Multiple Segments in Blazor Diagram Component | Syncfusion
+description: Checkout and learn here all about Multiple Segments in Syncfusion Blazor Diagram component and much more details.
+platform: Blazor
+control: Diagram Component
+documentation: ug
+---
+
+# Multiple segments in Blazor Diagram Component
+
+## How to create multiple segments
+
+Multiple segments can be defined one after another. To create a connector with multiple segments, define and add the segments to [connector.Segments](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.ConnectorSegment.html) collection. The following code example illustrates how to create a connector with multiple segments.
```

Comment:
Multiple segments can be defined one after another. To create a connector with multiple segments, define and add the segments to the [connector.Segments](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.ConnectorSegment.html) collection. The following code example illustrates how to create a connector with multiple segments.